app/controllers/flags_controller.rb

```diff
@@ -1,4 +1,5 @@
 class FlagsController < ApplicationController
+  include FlagsHelper
```

Comment:
(you shouldnt need to include it but also the flagshelper is empty so v confused :p